samples/TR/index.html

```diff
@@ -87,13 +87,8 @@
-	you can set a predefined page size (for example A4),
-	or switch to a spread view, to see two pages side by side.
```

Comment:
Why remove this? it seems like useful information to tell people about fixed sized pages, and at least until we implement `spread=auto` about how to switch to a spread view.